launch_ros/launch_ros/utilities/__init__.py

```diff
@@ -23,11 +23,15 @@
 from .normalize_remap_rule import normalize_remap_rule
 from .normalize_remap_rule import normalize_remap_rules
 from .to_parameters_list import to_parameters_list
+from .track_node_names import add_node_name
+from .track_node_names import get_node_name_count
 
 __all__ = [
     'evaluate_parameters',
     'normalize_parameters',
     'normalize_remap_rule',
     'normalize_remap_rules',
     'to_parameters_list',
+    'add_node_name',
+    'get_node_name_count'
```

Comment:
Alphasort

Reply:
There will be PEP8 F401 violations if these are alphasorted.

Reply:
is that caused by inter-module dependencies?

Reply:
Oops, sorry. It was because a comma was missing after alphasorting these lines.